packages/manager/apps/cloud/client/app/app.module.js

```diff
@@ -321,20 +321,6 @@ import './cloud/project/storage/storage.controller';
 import './cloud/project/storage/storage';
 import './common/cloud-main-controller.controller';
 import './common/phone.constant';
-import './dbaas/dbaas-metrics/dashboard/chart-pie/metrics-chart-pie.component';
-import './dbaas/dbaas-metrics/dashboard/metrics-dashboard.controller';
-import './dbaas/dbaas-metrics/header/metrics-header.controller';
-import './dbaas/dbaas-metrics/metrics-detail.controller';
-import './dbaas/dbaas-metrics/metrics.constant';
-import './dbaas/dbaas-metrics/metrics';
-import './dbaas/dbaas-metrics/metrics.service';
-import './dbaas/dbaas-metrics/platform/metrics-platform.controller';
-import './dbaas/dbaas-metrics/token/add/metrics-token-add.controller';
-import './dbaas/dbaas-metrics/token/delete/metrics-token-delete.controller';
-import './dbaas/dbaas-metrics/token/edit/metrics-token-edit.controller';
-import './dbaas/dbaas-metrics/token/metrics-token.controller';
-import './dbaas/dbaas-metrics/token/preview/metrics-token-preview.controller';
-import './dbaas/dbaas';
```

Comment:
I think we should keep it as long as `dbaas-logs` is not migrated (PR not merged yet : https://github.com/ovh/manager/pull/3771)